src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -103,17 +101,54 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging {
    * @return
    */
   def queryExecutionServiceForStatus()(implicit executionContext: ExecutionContext): Future[ExecutionServiceStatusResponse] = {
```

Comment:
The submission monitor now handles aborting submissions. "Queued" workflows will be marked as "aborted". "Submitted" and "running" workflows will be aborted through cromwell. Anything else will be left until they are in a more reliable state (i.e. a workflow with status "launching" won't be aborted until it reaches "submitted"). This is similar behavior to what the WorkspaceService.abortSubmission function did, but this handles more statuses